en/docs/deploy/elk-analytics-installation-guide.md

```diff
@@ -77,192 +77,13 @@ Follow the steps below to enable ELK-based analytics in WSO2 Identity Server.
 ### Install Filebeat
 1. [Install Filebeat](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation-configuration.html#installation) according to your operating system.
 
-2. Open the filebeat.yml file in the root directory and enter the following configurations.
-    ```
-    filebeat.inputs:
-    - type: filestream  
-      enabled: true
-      parsers:
-        - multiline:
-          type: pattern
-          pattern: '^[[:space:]]Event:'
-          negate: false
-          match: after
-      include_lines: ['Event:']
-      paths:
-      - <IS_HOME>/repository/logs/wso2carbon*.log
-    filebeat.registry.path: /var/lib/filebeat/registry
-    output.logstash:
-      hosts: ["localhost:5044"]
-    ```
+2. Open the **filebeat.yml** file in the root directory and enter the configurations found [here](https://github.com/wso2-extensions/identity-elk-integration/blob/main/filebeat/filebeat.yml).
```

Comment:
Instead of linking the term `here`, it is better to link `configurations`.

`Open the filebeat.yml file in the root directory and the [configurations().`

Reply:
ack. Will change.